descholar-ceo/AnnounceIT
# #170727279 User view specific announcements

#### What does this PR ​ do​ ?
As a user, I should be to view a specific announcement

#### Description of Task to be completed?
-User enters an id number of an announcement in a search bar
-User gets all announcements starting with the number he/she has typed so far

#### How should this be manually tested?
* Clone this repository in case you don't have it using this command `git clone https://github.com/descholar-ceo/AnnounceIT.git`
* If you use [VSCode](https://code.visualstudio.com/) as code editor, install [_live-server_](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer), and click on _Go Live_

* Click on enter button from homepage, 

* And then you will be redirected in the browser, here you will need to go to the root of the url you have in url-bar, and enter like `http://127.0.0.1:5500/UI/html/user-profile.html`, you will be redirected to the users page, or enter like this ``http://127.0.0.1:5500/UI/html/user-profile.html`,
* Enter any number you want in search bar, you will get results as you type

* Or if you don't use [VSCode](https://code.visualstudio.com/) as text editor, after cloning the repo, you navigate in `AnnounceIT/UI/html/` and you will see different files, you choose `user-profile.html` and right click and open with your preferred web browser

#### Any background context you want to provide?
* It would be better if you use  [VSCode](https://code.visualstudio.com/)

#### What are the relevant pivotal tracker stories?
[170727279](https://www.pivotaltracker.com/n/projects/2429498#)

#### Screenshots (​ if​ appropriate)
![Screenshot from 2020-01-20 01-23-47](https://user-images.githubusercontent.com/56886064/72690283-8a652500-3b23-11ea-8302-0ca9c9a88247.png)

#### Questions:
No Question
